src/main/java/solvd/laba/ermakovich/hf/kafka/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,56 @@
+package solvd.laba.ermakovich.hf.kafka;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.UUIDDeserializer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import reactor.kafka.receiver.KafkaReceiver;
+import reactor.kafka.receiver.ReceiverOptions;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@Slf4j
+@Configuration
+public class KafkaConsumerConfig {
+
+    @Value("${spring.kafka.consumer.bootstrap-servers}")
+    private String BOOTSTRAP_SERVERS;
```

Comment:
Only constants or ```private static final``` can use UPPER CASE in variable scope. Please follow simple java code conventions